tests/config-import.js

```diff
@@ -83,9 +123,8 @@ test('config import - icons', async () => {
   let config = await readConfig(
     './tests/fixtures/config/invalidIconFilename.zip'
   )
-  /* eslint-disable-next-line */
-  for await (const icon of config.icons()) {
-  }
+
+  await arrayFrom(config.icons())
```

Comment:
Minor: I'd move these to a separate PR, as it's not strictly related. But I don't feel strongly.